src/views/v2/tickets/item/extra.php

```diff
@@ -15,7 +15,8 @@
  * @version TBD
  *
  * @var Tribe__Tickets__Editor__Template $this
- * @var Tribe__Tickets__Ticket_Object    $ticket
+ * @var Tribe__Tickets__Ticket_Object $ticket
```

Comment:
If you see vars without a comment, let's try and catch those. Not a big deal if not, but if you're already in there making a change to add a var and you comment it, this could really help a person out when they're customizing.